### PR TITLE
Add link to line number 

### PR DESCRIPTION
**Pull request by RobinDev**
Permit to use link wich point directly to line number in docBlock : `/** @see src/example/test.php:12 */`

### CODE REVIEW

File: src/DocumentParser.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export class DocumentParser {
             withPositions: true
         },
         lexer: {
-            short_tags: true
+            shortTags: true
```
Comment (pwarchol) [inline src/DocumentParser.ts:19]: Why this change? In my experience, "shortTags" falg don't work.
Have you checked if lexer parses correctly files with short tags with this flag?
Please revert this line.

File: src/types.ts [hunk 2]
```diff
@@ -26,6 +28,7 @@ export class MyLink extends vscode.DocumentLink {
     document: vscode.TextDocument;
 
     constructor(range: vscode.Range, fileMatch: FileMatch, document: vscode.TextDocument, target?: vscode.Uri) {
+        console.log(target);
```
Comment (pwarchol) [inline src/types.ts:31]: Remove this console log

File: package-lock.json
Comment (pwarchol) [inline package-lock.json:1]: You shouldn't commit a new lock file, not everyone needs your packages :)
Please revert this file.

**Review by pwarchol — CHANGES_REQUESTED** [review]
Hi, code changes looks good, and i like this feature.
Need some minor corrections as described.

**Comment by RobinDev** [comment]
Ok fixed !

**Comment by pwarchol** [comment]
Thank you, new version released.
